ipfshttpclient/client/pin.py

```diff
@@ -67,11 +69,14 @@ def ls(self, type="all", **kwargs):
 		-------
 			dict : Hashes of pinned IPFS objects and why they are pinned
```

Comment:
Add the following above the *Returns* section to document the exception raised:

```
		Raises
		------
		~ipfsapi.exceptions.ErrorResponse
			A CID was given, that was not actually pinned
```

Reply:
Added.